docs/products/custom_domain.md

```diff
@@ -13,7 +13,7 @@ Required Items
 * CNAME edit access for a domain name
 
 
-### Domain Certificate
+## Domain Certificate
```

Comment:
This part of the cluster configuration. It should be nested as such.
```suggestion
### Domain Certificate
```

Reply:
Good point, ty